systemd-boot/add-kernel

```diff
@@ -12,7 +12,14 @@
 # Add kernel/initrd to boot config.
 
 err=0
-if [ -x /usr/bin/kernel-install ] ; then
+
+if [ -x /usr/bin/sdbootutil ] ; then
+  version="$1"
+  # kernel and initrd files are ignored, as sdbootutil is the one that
+  # will find the correct kernel file from the snapshots, and will
+  # create the correct initrd for the system
+  sdbootutil add-kernel "$version"
```

Comment:
Please use the '( set -x ; ... )' construction when running sdbootutil to get the command logged. Also, propagate an error to the $err var in case sdbootutil fails.

Reply:
yes, done